pyproject.toml

```diff
@@ -97,19 +97,38 @@ raw-options = { version_scheme = "release-branch-semver" }
 [tool.hatch.build.hooks.vcs]
 version-file = "fury/_version.py"
 
-[tool.blue]
+[tool.ruff]
 line_length = 88
 target-version = ["py38"]
-force-exclude = """
-(
-  _version.py
-  *docs/experimental/*
-)
-"""
-
-[flake8]
-max-line-length = 88
-extend-exclude = ["_version.py", "externals", "*docs/experimental*"]
+exclude = [
+    ".git",
+    "__pycache__",
+    "build",
+    "dist",
+    "_version.py",
+]
+
+[tool.ruff.lint]
+select = [
+  "F",
+  "E",
+  "C",
+  "W",
+  "B",
+  "I",
+]
+ignore = [
+  "E203"
+]
+
+[tool.ruff.lint.flake8-quotes]
+inline-quotes = "double"
+
+[tool.ruff.format]
+quote-style = "double"
```

Comment:
I am not sure about this. 

double quotes vs single quotes. For now, we were using single quotes in this project.

Reply:
`ruff` uses `double` by default, but I prefer to make that explicit.

Other projects:
- Nibabel: single: https://github.com/nipy/nibabel/blob/a2e5dee05cf374c22670ff9fd0d385ce366eb495/pyproject.toml#L141
- NumPy: does not enforce a quote style yet, contains both, e.g.: https://github.com/numpy/numpy/blob/970476737080a053d9be3d6f4479bb1b5cd17f19/tools/lint_diff.ini e.g. https://github.com/numpy/numpy/blob/970476737080a053d9be3d6f4479bb1b5cd17f19/numpy/ma/core.py
- SciPy does not enforce a quote style https://github.com/scipy/scipy/blob/main/tools/lint.toml, contains both e.g. https://github.com/scipy/scipy/blob/main/scipy/optimize/optimize.py
- Scikit-image: not sure, default `ruff` is double: https://github.com/scikit-image/scikit-image/blob/main/pyproject.toml#L154, but they have both in the code, e.g.
 https://github.com/scikit-image/scikit-image/blob/main/skimage/io/collection.py
https://github.com/scikit-image/scikit-image/blob/main/skimage/filters/_sparse.py
- Scikit-learn: not sure, default `ruff` is double: https://github.com/scikit-learn/scikit-learn/blob/main/pyproject.toml#L122 looks double quotes, e.g.
https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/compose/_column_transformer.py
https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/decomposition/_lda.py

Reply:
ok, let's go with double quotes